kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -454,6 +454,19 @@ public class KafkaRestConfig extends RestConfig {
           + "\"api.v3.clusters.*=1,api.v3.brokers.list=2\". A cost of zero means no rate-limit.";
   private static final String RATE_LIMIT_COSTS_DEFAULT = "";
 
+  public static final String STREAMING_CONNECTION_TIMEOUT_MS = "streaming.connection.timeout.ms";
```

Comment:
This went through about 6 names - it was hard to think of something that covered it without writing an essay.  I went with timout because that's a word we use a lot elsewhere, but I totally agree on the confusion.

`streaming.connection.max.duration.ms` was the previous version.  What do you think of that?